backend/molgenis-emx2-graphql/build.gradle

```diff
@@ -4,4 +4,6 @@ dependencies {
     implementation project(':backend:molgenis-emx2-tasks')
     implementation project(':backend:molgenis-emx2')
     implementation project(':backend:molgenis-emx2-datamodels')
+
+    testImplementation 'org.junit-pioneer:junit-pioneer:1.7.1'
```

Comment:
why

Reply:
removed. Was for testing lombok based classes but those are removed.